src/components/HomePage/BodyContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import styled from '@emotion/styled'
+import { neutral } from '@guardian/src-foundations/palette'
+
+import Container from '../Container'
+
+const StyledBodyContainer = styled(Container)`
+  border-top: ${neutral[46]} 0.2px solid;
+  border-left: ${neutral[46]} 0.2px solid;
+  border-right: ${neutral[46]} 0.2px solid;
```

Comment:
Why use a a border width of less than `1px`?

Reply:
This was done to match the styling on the Header component. Potentially, this has come from either the Figma board or Source, so a bit of work would be needed to determine why we've got sizes <1px